src/main/java/uk/gov/register/store/postgres/PostgresReadDataAccessLayer.java

```diff
@@ -0,0 +1,168 @@
+package uk.gov.register.store.postgres;
+
+import uk.gov.register.core.Entry;
+import uk.gov.register.core.Item;
+import uk.gov.register.core.Record;
+import uk.gov.register.db.*;
+import uk.gov.register.store.DataAccessLayer;
+import uk.gov.register.util.HashValue;
+
+import java.time.Instant;
+import java.util.*;
+import java.util.function.Function;
+
+public abstract class PostgresReadDataAccessLayer implements DataAccessLayer {
```

Comment:
As there's only one subclass of this, can it be combined with PostgresDataAccessLayer?